render.yaml

```diff
@@ -4,6 +4,16 @@ databases:
     user: hostedgpt
     plan: free
 
+envVarGroups:
+- name: generated-secrets
+  envVars:
+    - key: SECRET_KEY_BASE
+      generateValue: true
```

Comment:
From @krschacht's [original comment](https://github.com/hostedgpt/hostedgpt/pull/44#discussion_r1446820695):

> Oh, interesting! @robacarp how does this make it so we don’t have to copy that master.key value?

The rails credential file is new to me. The gist is, if deployment secrets are a management problem for your team, you can use an encrypted credential file to make it easier. In the case where you use the credentials file, rails warns:

<img width="749" alt="image" src="https://github.com/hostedgpt/hostedgpt/assets/208647/8702e0c0-11c3-49b4-a53d-393ecca3193d">

I've deleted the master.key file from our repo. I believe the intent with that file is that it would be copied post-deploy from some other file store, or otherwise hosted on long-living servers somewhere. With docker, we don't really have either of those anyway.

In this repo, our intent is to build a forkable project. Each incarnation of our project will have it's own credentials and they should be unique -- even declaring the credentials on our own from the start is a risk because people _don't change the defaults_. Render.com has a nice little override for this which generates a value for you when you stand up a server from a blueprint. A neat idea I've never come across, but I like it.

So I've also deleted the credentials file and replaced the rails master key with the only value which was in the credentials file, the secret key base.